.circleci/continue_config.yml

```diff
@@ -364,6 +451,42 @@ workflows:
                 - not: << pipeline.parameters.build_all >>
     build-and-test-all:
         jobs:
+            - build_ingress_nginx_module:
+                matrix:
+                    parameters:
+                        arch:
+                            - amd64
+                        version:
+                            - v1.11.3
+                            - v1.11.2
+                            - v1.11.1
+                            - v1.11.0
+                            - v1.10.4
+                            - v1.10.3
+                            - v1.10.2
+                            - v1.10.1
+                            - v1.10.0
```

Comment:
Would it be possible to build for v1.9.6 as well? This would make the transition easier, because it's the last version still supporting opentracing.

Reply:
Hey @robmp 
I don't quite understand the rationale behind building for pre v1.10. How would this help with the transition?

Reply:
We currently use opentracing with `datadog-collector-host` in our nginx ingress controllers, and we have many independently deployed ingresses which have opentracing enabled via the `enable-opentracing` annotation, and also have some additional opentracing configuration in `configuration-snippet` annotations.

Nginx ingress controller v1.10.0 drops support for opentracing, so in order to upgrade and switch to the datadog module, we would have to first remove all opentracing configuration from our ingresses, then upgrade the ingress controllers, then re-add the tracing configuration using the datadog module. This means we would have gaps in our tracing data during the transition, which is not ideal.

If the datadog module supported v1.9.6, we could switch the ingresses from opentracing to datadog one by one without any gaps in tracing data. The process would also be easier to test and roll back in case of any issues.

Reply:
Make sense, I added `v1.9.6` support in d9b8f77c06ed18b0867caca7f1d8b9a7dbff9cd0.

Reply:
Thank you!